### PR TITLE
fix: connections commands description fixed w/signoff

### LINKED ISSUE #11311: [mesheryctl] `exp connections` needs update in Long and short description.
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior
Currently the Short,Long and example are not appropriate. They don't make the command clear to the end user.
### Desired Behavior
The text inside the long, short and example should  be more user facing and helpful.
Update `docs/mesheryctlcommands/cmds.yml` as well for the commands.

### Screenshots/Logs

Screenshot from 2024-07-03 20-39-31

### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by rahulrana701**
**Notes for Reviewers**

This PR fixes #11311
here is the attached ss of the changes I made

Screenshot 2024-07-11 135056

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 430f4a65fa05706bd73548814a1ab712f656d7c1 at: https://668f9ce39a833618143b8bd3--meshery-docs-preview.netlify.app

File: mesheryctl/internal/cli/root/connections/connections.go [hunk 1]
```diff
@@ -19,11 +19,15 @@ var (
 
 var ConnectionsCmd = &cobra.Command{
 	Use:   "connections",
-	Short: "Manage connections",
-	Long:  `Manage connections`,
+	Short: "Manage Meshery connections",
+	Long:  `View and manage your Meshery connections`,
```
Comment (hexxdump) [inline mesheryctl/internal/cli/root/connections/connections.go:23]:
```suggestion
	Long:  `View and manage the Meshery connections`,
```

**Comment by Ashparshp** [comment]
@rahulrana701 Could we discuss this issue during the Meshery dev meeting call? Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

**Comment by lekaf974** [comment]
@rahulrana701 could you fix lint error please

**Comment by lekaf974** [comment]
> @rahulrana701 could you fix lint error please

forgot about this comment, just see that it looks not related to your changes

**Comment by l5io** [comment]
🚀 Preview for commit 2d7c7fee5e4ede6c1f20f124dff0fa781f05c9fc at: https://66cd9d0d215411c39bde0b8c--meshery-docs-preview.netlify.app

**Comment by rahulrana701** [comment]
@Ashparshp, sorry for the delay. All the checks have successfully passed, and this branch is good to go.

**Pull request merged**